app/templates/sell-quick-start.pug

```diff
@@ -52,7 +52,7 @@
               ul.uib-dropdown-menu.dropdown-menu.currency-dropdown(role="menu")
                 li(ng-repeat="currency in currencies"
                    role="menuitem"
-                   ng-click="changeSellCurrency({currency: currency}); getExchangeRate(); changeSymbol({currency: currency});"
+                   ng-click="changeSellCurrency({currency: currency}); getExchangeRate(); changeSymbol({currency: currency}); getQuote();"
```

Comment:
There's a lot happening in this one `ngClick`, could you consolidate these into a controller function?